src/styles/partials/components/_site-nav.scss

```diff
@@ -0,0 +1,161 @@
+/* Handles the widths and horizontal positioning for the slide-out menu and hamburger menu for desktop */
+@mixin set-navigation-width($width) {
+  #bc_site-nav .bc_site-nav__list-container {
+    width: $width;
+
+    &.invisible {
```

Comment:
More of question/ opening of discussion. Any reason using the invisible class vs selecting the parent div with `is-active`? Thinking this would accomplish the same functionality but with less markup, removing the need for the `invisible` class. In that case, I would imagine (speculating) the functionality of `.invisible` could be baked in to  `#bc_site-nav .bc_site-nav__list-container`. Thoughts?